includes/course-theme/class-sensei-course-theme-templates.php

```diff
@@ -499,5 +502,26 @@ private function should_hide_lesson_template( $post_type ) {
 		return false;
 	}
 
+	/**
+	 * Filters the block templates to hide the lesson template in the post editor if the course does not have learning mode enabled.
+	 *
+	 * @param array  $current_templates The block templates.
+	 * @param string $theme_name The theme name.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @internal
+	 *
+	 * @return array The filtered block templates.
+	 */
+	public function filter_single_lesson_template_in_learning_mode( $current_templates, $theme_name ) {
```

Comment:
Could this be a `private` function?

Reply:
Hi @donnapep ! 👋 Yap this actually is better suited to be a private function. I only declared it public with the internal tag to add tests for it easily.

Reply:
Ah, OK. Makes sense then. 🙂